src/components/theme/Widgets/__snapshots__/EmailWidget.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`EmailWidget renders an email view widget component 1`] = `
 <a
-  className="metadata email widget"
+  className="external metadata email widget"
   href="mailto:foo@bar.com"
```

Comment:
@sneridagh Do we want the external link icon for mailto and telephone links?

Reply:
I don't think so...

Reply:
I fixed this.